lib/publiccloud/utils.pm

```diff
@@ -79,6 +79,27 @@ sub register_addon {
     record_info('SUSEConnect time', 'The command SUSEConnect -r ' . get_addon_fullname($addon) . ' took ' . (time() - $cmd_time) . ' seconds.');
 }
 
+sub deregister_addon {
+    my ($remote, $addon) = @_;
+    my $arch = get_var('PUBLIC_CLOUD_ARCH', 'x86_64');
+    $arch = "aarch64" if ($arch eq "arm64");
+    my $timestamp = utc_timestamp();
+    record_info($addon, "Going to deregister '$addon' addon\nUTC: $timestamp");
+    my $cmd_time = time();
+    # ssh_remove_suseconnect_product($remote, $name, $version, $arch, $params, $timeout, $retries, $delay)
```

Comment:
leftover ?

Reply:
No. It's valid comment. The `registration:: ssh_remove_suseconnect_product()` has too many parameters.

Reply:
ah I see now , can you please move it two lines down so it would be clear to what it belongs

Reply:
Done.